README.md

```diff
@@ -85,34 +87,34 @@ cache @matchdefault {
-| `regex.exclude`                                   | The regex used to prevent paths being cached                                                   | `^[A-z]+.*$`                                                                                                            |
-| `stale`                                           | The stale duration                                                                             | `25m`                                                                                                                   |
-| `ttl`                                             | The TTL duration                                                                               | `120s`                                                                                                                  |
-| `log_level`                                       | The log level                                                                                  | `One of DEBUG, INFO, WARN, ERROR, DPANIC, PANIC, FATAL it's case insensitive`                                           |
+|  Key                    |  Description                                                                                                                                 |  Value example                                                                                                          |
+|:------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------------------|
+| `api`                   | The cache-handler API cache management                                                                                                       |                                                                                                                         |
+| `api.basepath`          | BasePath for all APIs to avoid conflicts                                                                                                     | `/your-non-conflict-route`<br/><br/>`(default: /souin-api)`                                                             |
+| `api.souin.enable`      | Enable the souin API with related routes                                                                                                     | `true`<br/><br/>`(default: false)`                                                                                      |
+| `api.souin.security`    | Enable JWT validation to access the resource                                                                                                 | `true`<br/><br/>`(default: false)`                                                                                      |
```

Comment:
The prometheus config should probably be in here as well.

---

As an aside though, Caddy already has prometheus metrics built in: https://caddyserver.com/docs/metrics

Would there be any way to integrate them with that, so there isn't two different metrics endpoints to have to hit, and two metrics stores in the same process? It feels like bloat to have them not be combined.